stable/nginx-ingress/values.yaml

```diff
@@ -5,7 +5,7 @@ controller:
   name: controller
   image:
     repository: gcr.io/google_containers/nginx-ingress-controller
-    tag: "0.9.0-beta.14"
+    tag: "0.9.0-beta.15"
```

Comment:
This image tag change is to something other than the title of the PR or the chart change. The corresponding changes appear to be against 2 different projects.